recipes/mypy-boto3-textract/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mypy-boto3-textract" %}
+{% set version = "1.35.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mypy_boto3_textract-{{ version }}.tar.gz
+  sha256: 8b43663513f0132a6be26d2f36d2435c71006c272a7714dec58dcc0da96dbe1f
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - boto3
+    - typing-extensions >=4.1.0  # [py<312]
```

Comment:
You could also make this an unconditional dependency and have a `noarch` package. That's simpler for maintenance.